src/runtime/graph/debug/graph_runtime_debug.cc

```diff
@@ -119,6 +152,14 @@ PackedFunc GraphRuntimeDebug::GetFunction(
           this->DebugGetNodeOutput(args[0], args[1]);
         }
       });
+  } else if (name == "run_individual") {
```

Comment:
Could you expose this function to python API and add a test case?

Reply:
Yes, will do